outmon/solvermon.go

```diff
@@ -413,3 +414,14 @@ func (sm *SolverMonitor) reprintFailure(errVertex *vertexMonitor, phaseText stri
 	}
 	errVertex.printError()
 }
+
+
+func (sm *SolverMonitor) modifyId(id string) string {
+	modifiedId := id
+	for old, new := range map[string]string{
```

Comment:
redefining this map on each call seems inefficient, it should be global and created once via go's `init()` function.